modules/logging-release-notes-6-0-0.adoc

```diff
@@ -0,0 +1,97 @@
+// module included in logging-6-0-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-6-0-0_{context}"]
+= Logging 6.0.0
+
+This release includes link:https://access.redhat.com/errata/RHBA-2024:6693[OpenShift Logging Bug Fix Release 6.0.0]
+
+[id="logging-release-notes-6-0-0-compatibility-support-matrix"]
+= Compatibility and support matrix
+
+In the table, components are marked with the following statuses:
+
+[horizontal]
+TP:: Technology Preview
+GA:: General Availability
+
+The components or features in link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] status are experimental and are not intended for production use.
+
+[NOTE]
+====
+To know about compatible {product-title} versions for {logging-uc} releases, see link:https://access.redhat.com/product-life-cycles?product=Red%20Hat%20OpenShift%20Logging[Product Life Cycles].
+====
+
+.Components support matrix
+[options="header"]
+|===
+
+| {logging-uc} Version 6+| Component Version
+
+| Operator | `eventrouter` | `logfilemetricexporter` | `loki` | `lokistack-gateway` | `opa-openshift` | `vector`
+
+|6.0 | 0.4 (GA) | 1.1 (GA) | 3.1.0 (GA) | 0.1 (GA) | 0.1 (GA) | 0.37.1 (GA)
+
+|===
+
+[id="logging-release-notes-6-0-0-removal-notice"]
+== Removal notice
+
+* With this release, logging no longer supports the `ClusterLogging.logging.openshift.io` and `ClusterLogForwarder.logging.openshift.io` custom resources. Refer to the product documentation for details on the replacement features. (link:https://issues.redhat.com/browse/LOG-5803[LOG-5803])
+
+* With this release,logging no longer manages or deploys log storage (such as Elasticsearch), visualization (such as Kibana), or Fluentd-based log collectors. (link:https://issues.redhat.com/browse/LOG-5368[LOG-5368])
+
+[NOTE]
+====
+In order to continue to use Elasticsearch and Kibana managed by the elasticsearch-operator, the administrator must modify those object's ownerRefs before deleting the ClusterLogging resource.
```

Comment:
Is this a supported solution?

Reply:
I would let engineering confirm that. If there are comments to remove this solution, I will.

Reply:
Removing the owner-reference is part of the recommended migration to using LokiStack as log storage. It allows the customer to continue to run ES & Kibana to query logs still stored there until the retention period has passed.